src/cli/telemetry/test/telemetry.spec.ts

```diff
@@ -3,12 +3,21 @@ import { mockValidatedConfig } from '@stencil/core/testing';
 
 import { createConfigFlags } from '../../../cli/config-flags';
 import { DIST, DIST_CUSTOM_ELEMENTS, DIST_HYDRATE_SCRIPT, WWW } from '../../../compiler/output-targets/output-utils';
+import * as environment from '../../../compiler/sys/environment';
 import { createSystem } from '../../../compiler/sys/stencil-sys';
 import type * as d from '../../../declarations';
 import * as shouldTrack from '../shouldTrack';
 import * as telemetry from '../telemetry';
 import { anonymizeConfigForTelemetry } from '../telemetry';
 
+const browserEnvGetter = jest.fn().mockReturnValue(false);
+
+Object.defineProperty(environment, 'IS_BROWSER_ENV', {
+  get() {
+    return browserEnvGetter();
+  },
+});
```

Comment:
tbh I couldn't find a better way to mock out an exported constant value 🤷‍♀️ if you know of one I am all ears!

Reply:
I'm not sure of how to do one like this either. Seems straightforward enough to me

Reply:
Jest doesn't leave these hanging around right? I.E. what happens to `environment` when this test suite goes out of scope? I don't remember TBH.

Reply:
I believe it cleans up after itself